int32[]   motor_id
float32[] position_setpoint
float32[] velocity
---
bool    success
float32 position
float32 velocity
